lib/util/index.ts

```diff
@@ -1,21 +1,13 @@
 import * as errors from 'balena-errors';
 import cloneDeep = require('lodash/cloneDeep');
-import throttle = require('lodash/throttle');
-import * as memoizee from 'memoizee';
-import * as moment from 'moment';
 import * as Pine from '../../typings/pinejs-client-core';
-import { getOsUpdateHelper as updateHelper } from './device-actions/os-update';
-import * as dt from './device-types';
 
 export interface ErrorResponse {
 	code: string;
 	statusCode?: number;
 	body?: string;
 }
 
-export const getOsUpdateHelper = updateHelper;
-export const deviceTypes = dt;
```

Comment:
These re-exports were convenient, but also only used for very small % of the files/methods.